main.py

```diff
@@ -196,10 +200,36 @@ def run_main():
 elif len(sys.argv) == 1:
     # the name of the executed file always gets passed
     run_main()
+
+elif len(sys.argv) == 4 and sys.argv[1] == '--new-token' and sys.argv[2] == '--file' and sys.argv[3]:
+    with open(sys.argv[3]) as json_file:
+        file_result = json.load(json_file)
+        email_value = file_result['email']
+        password_value = file_result['password']
+    
+        if email_value and password_value:
+            run_new_token(email_value, password_value)
+        else:
+            run_new_token()       
```

Comment:
I'd like some sort of error / hint that is telling me how my call was wrong, as it might otherwise be confusing to just get prompted to enter the credentials after you tried to pass them as CLI args.

Also, how would this behave when called from a cron job? Might it result in indefinitely hanging processes that are waiting for input if the user messes up their crontab config?

Reply:
I'll implement calls to `sys.exit()` as failures are fatal.